src/http-server.ts

```diff
@@ -19,40 +31,40 @@ export class HttpServer {
     if (this.headersTimeoutMillis) {
       this.http.headersTimeout = this.headersTimeoutMillis;
     }
-      
+
     // This event is emitted when a new TCP stream is established
     this.http.on('connection', socket => {
-      
+
       // set socket to idle. this same socket will be accessible within the `http.on('request', (req, res))` event listener
       // as `request.connection`
-      socket.__idle = true;
+      socket['__idle'] = true;
```

Comment:
you could use a `Symbol` to do this in a more hidden way

Reply:
ah! because symbol properties are not enumerable?

Reply:
technically they are enumerable, just not via `for ( ... in ... )` or `Object.keys`/`Object.entries`.  you have to use `Object.getOwnPropertySymbols` to grab them, which most dont bother with

the main benefit of `Symbol` is that it's impossible to fetch unless you have that specific instance of the `Symbol` (i.e. `Symbol("test") !== Symbol("test")`), so unless you have a reference to it (or use `Object.getOwnPropertySymbols`) then there's no way to fetch its corresponding value

this allows for you to basically "hide" things on other objects (again not counting `Object.getOwnPropertySymbols`) that will not appear in any of the normal/common paths :)

Reply:
Awesome thanks for the explanation. Will apply this in `ws-server.ts` as well for `isIdle`

Reply:
done